### PR TITLE
fix(ports): multiple domain userstyles are correctly formatted

### PR DESCRIPTION
**Pull request by isabelroses**
closes https://github.com/catppuccin/website/issues/43

### CODE REVIEW

File: src/components/cards/Port.astro [hunk 1]
```diff
@@ -17,7 +17,7 @@ const { title, link, categories, color } = Astro.props;
 
 <a href={link} class="port-card">
   <div class="port-header">
-    <p class="port-name">{title}</p>
+    <p class="port-name">{Array.isArray(title) ? title.join(", ") : title}</p>
```
Comment (uncenter) [inline src/components/cards/Port.astro:20]: I believe we separated with a slash over at userstyles.

Comment (isabelroses) [reply src/components/cards/Port.astro:20]: Not on the rich display.
image

Comment (uncenter) [reply src/components/cards/Port.astro:20]: Right. Is that something we should change potentially? The linter is requiring titles to be using slashes as separators for the title, and commas/ands for the descriptions.

Comment (isabelroses) [reply src/components/cards/Port.astro:20]: > Right. Is that something we should change potentially? The linter is requiring titles to be using slashes as separators for the title, and commas/ands for the descriptions.

Probably best to continue this discussion on the userstyles repo or on the discord server.

**Review by sgoudham — APPROVED** [review]
As discussed on discord, types don't provide a whole lot of confidence here but we'll achieve this confidence once we start validating against the schema :+1: 

Thanks for fixing!

**Pull request merged**